boards/silabs/radio_boards/siwx917_rb4338a/doc/index.rst

```diff
@@ -39,6 +39,12 @@ Programming and Debugging
 
 .. zephyr:board-supported-runners::
 
+Wi-Fi Features
+==============
+
+Refer to the :ref:`siwx917_wifi_features` page for a list of supported Wi-Fi features.
```

Comment:
I believe this section should be placed (or merged) with `Supported Features`, a bit above.